src/video/uikit/SDL_uikitmodes.m

```diff
@@ -30,21 +30,31 @@
 
 @implementation SDL_UIKitDisplayData
 
+#if !TARGET_OS_XR
 - (instancetype)initWithScreen:(UIScreen *)screen
 {
     if (self = [super init]) {
         self.uiscreen = screen;
     }
     return self;
 }
+#endif
 
+- (instancetype) init
```

Comment:
Don't we need `[super init]` here? Is it possible to just omit this and rely on the base class init?